docs/reference/eql/eql-search-api.asciidoc

```diff
@@ -56,9 +56,26 @@ To search all data streams and indices in a cluster, use
 [[eql-search-api-query-params]]
 ==== {api-query-parms-title}
 
-include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
+`allow_no_indices`::
+(Optional, boolean)
++
+NOTE: This parameter's behavior differs from the `allow_no_indices` parameter
+used in other <<multi-index,multi-target APIs>>.
++
+If `false`, any wildcard expression, <<indices-aliases,index alias>>, or `_all`
+value that targets only missing or closed indices returns an error. This applies
+even if the request targets other open indices. For example, if an index starts
+with `foo` but no index starts with `bar`, a request targeting `foo*,bar*`
+returns an error.
 +
-Defaults to `false`.
+If `true`, only requests that exclusively target missing or closed indices
+return an error. If the request targets any open index, it does not return an
+error. For example, if an index starts with `foo` but no index starts with
+`bar`, a request targeting `foo*,bar*` does not return an error. However, a
+request that targets only `bar*` still returns an error.
++
```

Comment:
```suggestion
If `true`, requests that exclusively target missing or closed indices
return an error, unless the request targets an open index.
For example, a request targeting `foo*,bar*` does not return an error
if any index starts with `foo`, but no index starts with
`bar`. However, a request that targets only `bar*` still returns an error.
+
```

Reply:
I think that combining the second sentence into the first helps to highlight the conditional statement.